pkg/lint/lintersdb/manager.go

```diff
@@ -522,6 +524,11 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 			WithLoadForGoAnalysis().
 			WithURL("https://github.com/Antonboom/nilnil").
 			WithSince("v1.43.0"),
+		linter.NewConfig(golinters.NewTenv(tenvCfg)).
+			WithSince("v1.43.0").
+			WithPresets(linter.PresetPerformance, linter.PresetStyle).
```

Comment:
Is there any performance improvement in this linter pointed out?

Reply:
thanks, this is mistook.
I fixed it.